### DISCUSSION TITLE
How to send a scheduled SMS?

### DISCUSSION THREAD

**Discussion by githubfollow** [caioagiani/mobizon-node#59] [Q&A] [answered]
Hello, I would like to know if it is possible to send a scheduled SMS?

**Comment by caioagiani** [comment, marked as answer]
@githubfollow Hi!

Yes, follow this params on PR #51 

```js
const sendSmsScheduled = await mobizon.sendSms({
  recipient: process.env.NUMBER,
  from: '',
  text: 'SMS sent by Mobizon (scheduled).',
  params: {
    name: 'SMS Scheduled',
    deferredToTs: '2022-01-26 01:00:00',
  },
});

console.log(sendSmsScheduled);
```

**Reply by githubfollow** [reply]
> @githubfollow Hi!
> 
> Yes, follow this params on PR #51
> 
> ```js
> const sendSmsScheduled = await mobizon.sendSms({
>   recipient: process.env.NUMBER,
>   from: '',
>   text: 'SMS sent by Mobizon (scheduled).',
>   params: {
>     name: 'SMS Scheduled',
>     deferredToTs: '2022-01-26 01:00:00',
>   },
> });
> 
> console.log(sendSmsScheduled);
> ```

Thanks!
